content/search-github/searching-on-github/searching-issues-and-pull-requests.md

```diff
@@ -102,6 +102,7 @@ The `author` qualifier finds issues and pull requests created by a certain user
 | <code>author:<em>USERNAME</em></code> | [**cool author:gjtorikian**](https://github.com/search?q=cool+author%3Agjtorikian&type=Issues) matches issues and pull requests with the word "cool" that were created by @gjtorikian.
 | | [**bootstrap in:body author:mdo**](https://github.com/search?q=bootstrap+in%3Abody+author%3Amdo&type=Issues) matches issues written by @mdo that contain the word "bootstrap" in the body.
 | <code>author:app/<em>USERNAME</em></code> | [**author:app/robot**](https://github.com/search?q=author%3Aapp%2Frobot&type=Issues) matches issues created by the integration account named "robot."
+| <code>-author:<em>USERNAME</em></code> | [**-author:app/robot**](https://github.com/search?q=-author%3Aapp%2Frobot&type=Issues) matches issues created by any user other than the account named "robot." The minus sign, or dash character (<code>-</code>) before the qualifier signifies a logical NOT for the qualifier in the search query.
```

Comment:
Thanks for this suggestion and for following the style of the other examples ✨ 

Returning to the issue you opened, I read:

> I was specifically trying to find out how to search pull requests for everything NOT from a specific author. I didn't find anything in this article already (or an existing issue) with the tip to add the dash before the qualifier. I don't think it warrants its own section of the article, just maybe a single example, like in the Search by author section, showing how to do the logical NOT search.

I'm wondering if this might be easier for people to find if we add the information to the tips at the start of the article (lines 22-26). This is where we remind people that they can quote multi-term keywords.